components/public-api-server/pkg/webhooks/stripe.go

```diff
@@ -110,6 +111,25 @@ func (h *webhookHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
+	case CustomerUpdatedEventType:
+		logger.Info("Handling CustomerUpdatedEventType")
+		customerID, ok := event.Data.Object["id"].(string)
+		if !ok {
+			logger.Error("Failed to identify customer ID from Stripe webhook.")
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+
+		if _, ok := event.Data.PreviousAttributes["address"]; ok {
```

Comment:
Ah, and we don't need to listen for "VAT ID changed" because no matter what, we are always only interested in their address. :+1: